plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -331,11 +341,36 @@ func (e *quotaEvaluator) checkRequest(quotas []api.ResourceQuota, a admission.At
 		return quotas, nil
 	}
 
+	// if we have limited resources enabled for this resource, always calculate usage
+	inputObject := a.GetObject()
+	mustHaveCoveringQuota := []api.ResourceName{}
+	for _, limitedResource := range e.config.LimitedResources {
+		limitedGroupResource := schema.GroupResource{Group: limitedResource.APIGroup, Resource: limitedResource.APIResource}
```

Comment:
This happens on every API request if I turn these on?  Compare without allocating?